packages/@react-aria/utils/src/constants.ts

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright 2024 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+// Custom event names for updating the autocomplete's aria-activedecendant.
+export const CLEAR_FOCUS_EVENT = 'react-aria-clear-focus';
+export const FOCUS_EVENT = 'react-aria-focus';
+export const UPDATE_ACTIVEDESCENDANT = 'react-aria-update-activedescendant';
```

Comment:
I had considered that but it felt weird to import that package into `@react-aria/autocomplete` when there isn't anything selection related going on per say. If anything, I wish there was a even lower level hook than useSelectableCollection/useSelectableItem to put all this virtual focus stuff in since they aren't explicitly selection related. 

Happy to move these into `@react-aria/selection` based on what people think but it just felt like `@react-aria/utils` was broader place to store these for now